graph/ui-tooltips/src/lib/atomizer-tooltip.tsx

```diff
@@ -26,19 +26,19 @@ export function AtomizerTooltip(props: AtomizerTooltipProps) {
             href="https://nx.dev/ci/features/split-e2e-tasks"
             text="automatically split"
           />
-          {
-            ' this potentially slow task into separate tasks for each file. We recommend enabling '
-          }
-          {!props.connectedToCloud && (
-            <>
-              <Link href="https://nx.app/" text="Nx Cloud" />
-              {' and '}
-            </>
+          {' the potentially slow'}
+          <code className="mx-2 inline rounded bg-gray-100 px-1 font-mono text-gray-800 dark:bg-gray-700 dark:text-gray-300">
```

Comment:
Minor: <code> is already inline, I don't think we need `inline` class name here.